src/less/workingfiles/submit.less

```diff
@@ -342,15 +295,21 @@ th {
 
 .additional-data-wrapper{
     width: 12rem; /*220px;*/
-    padding-left:25%; /*37px;*/
+    padding-left:3.6rem; /*37px;*/
 }
 .facet-additional-input {
     .submit-input-box();
     font-size: 8.5pt;
     margin-top: 5px;
     width: 12rem; /*200px;*/
-    margin-left: -45px; /*45px;*/
+    //margin-left: -45px; /*45px;*/
 }
+.facet-additional-input::-webkit-scrollbar {
+        display: none;
+        z-index:1;
+        position: relative;
+        //not working for mozilla yet
```

Comment:
After a quick google Firefox seems rekt, do you how IE, Edge or Safari handles this stuff?